contracts/DependencyRegistryV0.sol

```diff
@@ -33,8 +33,8 @@ contract DependencyRegistryV0 is
     OwnableUpgradeable,
     IDependencyRegistryV0
 {
-    using BytecodeStorage for string;
-    using BytecodeStorage for address;
+    using BytecodeStorageWriterV1 for string;
+    using BytecodeStorageWriterV1 for address;
```

Comment:
we don't actually need to attach this to address anymore 🙂 (this comment applies to all contracts in this PR)